plugins/modules/ndo_fabric_span_session_source.py

```diff
@@ -0,0 +1,752 @@
+    - The name of the SPAN Session source.
+    type: str
+  direction:
+    description:
+    - The direction of the SPAN Session source.
+    - Defaults to O(direction=incoming) when unset during creation.
+    type: str
+    choices: [ incoming, outgoing, both ]
+  span_drop_packets:
+    description:
+    - The SPAN Drop Packets of the SPAN Session source.
+    - Defaults to O(span_drop_packets=false) when unset during creation.
+    - The O(filter_epg) and O(filter_l3out) is not configurable when this parameter O(span_drop_packets=true) set to true.
+    type: bool
+  filter_epg:
+    description:
+    - The Filter EPG of the SPAN Session source.
+    - When the Filter EPG is specified in the configuration, the Filter L3Out will be removed.
+    - Filter L3Out and Filter EPG cannot be configured simultaneously.
+    - Providing an empty dictionary O(filter_epg={}) will remove the filter l3out from the SPAN Session source.
+    type: dict
+    suboptions:
+      uuid:
+        description:
+        - The UUID of the EPG used to configure the Filter EPG.
+        - This parameter or O(filter_epg.epg) is required.
+        type: str
+        aliases: [ epg_uuid ]
+      epg:
+        description:
+        - The EPG object detail used to configure the Filter EPG.
+        - This parameter or O(filter_epg.uuid) is required.
+        type: dict
+        suboptions:
+          name:
+            description:
+            - The name of the EPG.
+            type: str
+            required: true
+          template:
+            description:
+            - The name of the template that contains the EPG.
+            - This parameter or O(filter_epg.epg.template_id) is required.
+            type: str
+          template_id:
+            description:
+            - The ID of the template that contains the EPG.
+            - This parameter or O(filter_epg.epg.template) is required.
+            type: str
+          schema_id:
+            description:
+            - The ID of the schema that contains the EPG.
+            - This parameter or O(filter_epg.epg.schema) is required.
+            type: str
+          schema:
+            description:
+            - The name of the schema that contains the EPG.
+            - This parameter or O(filter_epg.epg.schema_id) is required.
+            type: str
+          anp:
+            description:
+            - The name of the ANP that contains the EPG.
+            - This parameter or O(filter_epg.epg.anp_uuid) is required.
+            type: str
+          anp_uuid:
+            description:
+            - The UUID of the ANP that contains the EPG.
+            - This parameter or O(filter_epg.epg.anp) is required.
+            type: str
+  filter_l3out:
+    description:
+    - The Filter L3Out of the SPAN Session source.
+    - When the Filter L3Out is specified in the configuration, the Filter EPG will be removed.
+    - Filter L3Out and Filter EPG cannot be configured simultaneously.
```

Comment:
so we are adding this line now as standard? I have made a note to my open PR

Reply:
We do not have a standard sentence for mutual exclusives condition. But changed the current description to match the current format.